Windowsでのmake setupが失敗する

続いて、`make run` の実行でエラーが発生

```sh
❯❯❯ make run
crystal run src/nes.cr
Showing last frame. Use --error-trace for full trace.

In lib/stumpy_png/src/stumpy_png.cr:1:1

 1 | require "zlib"
     ^
Error: can't find file 'zlib'

If you're trying to require a shard:
- Did you remember to run `shards install`?
- Did you make sure you're running the compiler in the same directory as your shard.yml?
make: *** [Makefile:5: run] Error 1
```

表記方法が変わっているらしい。これもライブラリのアップデートで直りそう
https://github.com/stumpycr/stumpy_png/issues/35

`make setup` からの `make run` で HELLO, WORLD の画像が出力できた

--- a/shard.yml
+++ b/shard.yml
@@ -10,7 +10,7 @@ description: |
 dependencies:
   stumpy_png:
     github: stumpycr/stumpy_png
-    version: "~> 4.5"
+    version: "~> 5.0.1"
 
 development_dependencies:
   ameba:

--- a/shard.yml
+++ b/shard.yml
@@ -15,6 +15,6 @@ dependencies:
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: ~> 0.12.0
+    version: ~> 1.0.0
 
 license: MIT

